docs/governance/staking-with-realms.md

```diff
@@ -1,82 +1,69 @@
 ---
-id: staking-with-realms
-title: Staking veHNT on Realms
-pagination_label: Realms
-sidebar_label: Staking veHNT on Realms
-description: Helium on Realms Documentation
+id: staking-with-helium-vote
+title: Staking on Helium Vote
+pagination_label: Helium Vote
+sidebar_label: Staking on Helium Vote
+description: Helium on Helium Vote Documentation
 image: https://docs.helium.com/img/link-image.png
-slug: /governance/staking-with-realms
+slug: /governance/staking-with-helium-vote
 ---
 
 import useBaseUrl from '@docusaurus/useBaseUrl'
 
-<img className="docsheader" src={useBaseUrl('/img/blockchain/realmsheader.png')} />
-
-Like many other Solana Ecosystem projects, Helium utilizes [Realms](https://realms.heliumvote.com/)
-to organize and manage vote escrow tokens, delegation, and more. Stakes can be created and later
-managed within Realms at any time at https://realms.heliumvote.com/.
+Helium utilizes [Helium Vote](https://heliumvote.com), a native application built by the Helium 
+Foundation to organize and manage vote escrow tokens, delegation, and more. Stakes can be created 
+and later managed within Helium Vote at any time at https://heliumvote.com.
 
 ## Creating a Stake
 
-Anyone with HNT in a wallet can create a stake using Realms either using an in-wallet browser or a
-browser-connected wallet.
+Anyone with HNT, MOBILE or IOT in their Helium Wallet App or Solana-compatible browser wallet
+can be used (such as Phantom orBackpack).
```

Comment:
sentence ends weird. consider rewriting as 2 sentences

Reply:
Updated
Anyone with HNT, MOBILE, or IOT in their Helium Wallet App or Solana-compatible browser wallet
(such as Phantom orBackpack) can stake.